server/constants/notificationEvents.ts

```diff
@@ -1,13 +1,12 @@
 import { NotificationType } from '../data/orchestrationApiTypes'
 
-export const notificationTypes: Readonly<Record<NotificationType, string>> = {
+// export const notificationTypes: Readonly<Record<NotificationType, string>> = {
```

Comment:
A few extra types came through here that I don't think we're using. Rather than add more marked `TODO` I've made the type `Partial` so it doesn't complain. Once we support all the available types we can remove this.